docs/config-options-listings.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: config-options-listings
+title: Listings For All BuildBuddy Options
+sidebar_label: Options Listings
```

Comment:
Maybe we should call this "All Options" for the id, title, and sidebar label (you'll need to change it in sidebars.js too)